README.md

```diff
@@ -183,6 +184,7 @@ The programatic APIs take a configuration object very equivalent to the CLI opti
 - `header` - Boolean, determines whether or not CSV file will contain a title column. Defaults to `true` if not specified.
 - `includeEmptyRows` - Boolean, includes empty rows. Defaults to `false`.
 - `withBOM` - Boolean, with BOM character. Defaults to `false`.
+- `locale` - String, locale to use for numbers formatting, Defaults to no locale formatting. 
```

Comment:
Would need to have a way to know what options are allowed, maybe a link to a list?